example.py

```diff
@@ -66,53 +66,97 @@ def load(
 def main(
     ckpt_dir: str,
     tokenizer_path: str,
-    temperature: float = 0.8,
-    top_p: float = 0.95,
+    temperature: float = 0.7,
+    # top_p: float = 0.95,
+    top_p: float = 0.0,
+    top_k: int = 40,
+    repetition_penalty: float = (1 / 0.85),
     max_seq_len: int = 512,
+    max_gen_len: int = 256,
     max_batch_size: int = 32,
+    seed: int = 1,
```

Comment:
Hey! Good code!
Just one thing, _seed_ is supposed to be a constant. So I would suggest to create that variable outside the parameter of the function: ```RANDOM_SEED=1```.